src/studio-header/MobileHeader.jsx

```diff
@@ -106,8 +110,17 @@ class MobileHeader extends React.Component {
               </Menu>
             ) : null}
         </div>
-        <div className="w-100 d-flex justify-content-center">
+        <div className="w-100 d-flex align-items-center">
           {logoDestination === null ? <Logo className="logo" src={logo} alt={logoAltText} /> : <LinkedLogo className="logo" {...logoProps} itemType="http://schema.org/Organization" />}
+          <a
+            className="course-title-lockup"
+            style={{ lineHeight: 1 }}
+            href={courseTitleDestination}
+            aria-label={intl.formatMessage(messages['header.label.courseOutline'])}
+          >
+            <span className="d-block small m-0" data-test-id="course-org-number">{courseOrg} {courseNumber}</span>
+            <span className="d-block m-0 font-weight-bold" data-test-id="course-title">{courseTitle}</span>
+          </a>
```

Comment:
Not sure if course lockup should be its own component, because this is the same code being used in two places.

Reply:
I didn't create a new component, but I did move the lockup code to the Header component, which then passes it to both the Mobile and Desktop header.